README.md

```diff
@@ -2,11 +2,8 @@
 
 A Dota 2 bot.
 
-This version of the code creates a lobby, waits for a player to join, selects Nature's Prophet, and
-makes a move towards the fountain. Past incarnations of this project have been used to make a
-spectactor site similar to TrackDOTA with real time updates. There's a few critical pieces I've
-decided not to share and it wont work without fixes. The rest of the code is probably mostly
-working, I haven't touched it in a while. Sorry for the dragons.
+This version of the code creates a lobby, ~~waits for a player to join, selects Nature's Prophet, and
+makes a move towards the fountain~~ (this part is broken, because can't get correct IP address, if you want to fix this, start your investigation from ```string[] split = client.Lobby.connect.Split(':');```).
+Past incarnations of this project have been used to make a spectactor site similar to TrackDOTA with real time updates. There's a few critical pieces I've decided not to share and it wont work without fixes. The rest of the code is probably mostly working, I haven't touched it in a while. Sorry for the dragons.
```

Comment:
Since you're touching these lines do you mind updating the part about

"There's a few critical pieces I've decided not to share and it wont work without fixes. The rest of the code is probably mostly working, I haven't touched it in a while."

And give it a 2024 status update? I forgot to update the README when I added the auth ticket so those pieces are now included.